### PR TITLE
Improve logging text in config.sample.php

### PR DESCRIPTION
**Pull request by mmattel**
## Description
This PR just improves the text in the logging section of config.sample.php a bit and references for the details the logging section in the documentation - keeping config.sample.php small and clean.

The log rotate script will be added in the logging section at the documentation!

## Related Issue

- Fixes https://github.com/owncloud/documentation/issues/2998 
(Document best practice examples to rotate/compress oC logfiles)

## Motivation and Context
Improve log rotation capabilities, see referenced doc issue

## How Has This Been Tested?

No test necessary, doc text changes only.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [x] Documentation ticket raised: https://github.com/owncloud/documentation/issues/2998

## Open tasks:

- [x] Backport (if applicable set "backport-request" label and remove when the backport was done)

@voroyam @michaelstingl

### CODE REVIEW

**Comment by mmattel** [comment]
Drone tells...
```
Error response from daemon: Get https://registry-1.docker.io/v2/selenium/standalone-chrome-debug/manifests/latest: 
dial tcp: lookup registry-1.docker.io on 172.16.102.1:53: no such host
```

File: config/config.sample.php [hunk 1]
```diff
@@ -606,6 +606,11 @@
 
 /**
  * Logging
+ * 
+ * These parameters configure the options for logging.
```
Comment (settermjd) [inline config/config.sample.php]: I'd suggest making it just a bit shorter and more direct with:

> These parameters configure the logging options.

File: config/config.sample.php [hunk 2]
```diff
@@ -606,6 +606,11 @@
...
+ * For additional information or advanced configuration please see the logging
```
Comment (settermjd) [inline config/config.sample.php]: One minor, nitpicky, change:

> For additional information or advanced configuration, please see the logging section in the documentation.

File: config/config.sample.php [hunk 3]
```diff
@@ -694,11 +699,13 @@
...
- * Enables log rotation and limits the total size of logfiles. The default is 0,
- * or no rotation. Specify a size in bytes, for example 104857600 (100 megabytes
- * = 100 * 1024 * 1024 bytes). A new logfile is created with a new name when the
- * old logfile reaches your limit. If a rotated log file is already present, it
- * will be overwritten.
+ * Enables log rotation and limits the total size of the logfiles. 
+ * The default is 0 or false which disables log rotation. 
+ * Specify a size in bytes, for example 104857600 
+ * (100 megabytes = 100 * 1024 * 1024 bytes). 
+ * A new logfile is created with a new name when the old logfile reaches your limit.
```
Comment (settermjd) [inline config/config.sample.php]: s/when the old logfile reaches your limit./when the old logfile reaches the defined limit./g

File: config/config.sample.php [hunk 4]
```diff
@@ -694,11 +699,13 @@
...
- * = 100 * 1024 * 1024 bytes). A new logfile is created with a new name when the
- * old logfile reaches your limit. If a rotated log file is already present, it
- * will be overwritten.
+ * Enables log rotation and limits the total size of the logfiles. 
+ * The default is 0 or false which disables log rotation. 
+ * Specify a size in bytes, for example 104857600 
+ * (100 megabytes = 100 * 1024 * 1024 bytes). 
+ * A new logfile is created with a new name when the old logfile reaches your limit. 
+ * If a rotated log file is already present, it will be overwritten.
+ * If enabled, only the active log file and one rotation file are present.
```
Comment (settermjd) [inline config/config.sample.php]: s/and one rotation file are present./and one rotated file are stored./g

**Review by settermjd — CHANGES_REQUESTED** [review]
On the whole, it looks good. Just a few minor changes.

**Comment by mmattel** [comment]
Changes added, many thanks @settermjd

**Review by settermjd — APPROVED** [review]
Thanks!

**Comment by mmattel** [comment]
I love drone...
```
Error response from daemon: Get https://registry-1.docker.io/v2/: net/http: request canceled 
while waiting for connection (Client.Timeout exceeded while awaiting headers)
```

**Comment by phil-davis** [comment]
I restarted the drone job for you. Fingers-crossed.

**Comment by phil-davis** [comment]
Backport ``stable10`` #32049

**Pull request merged**